calour/heatmap/plotgui_jupyter.py

```diff
@@ -178,7 +179,9 @@ def show_info(self):
                 annt_type = details.get('annotationtype', 'None')
                 # annt_id = details.get('annotationid', 'NA')
                 ccolor = colors.get(annt_type, 'black')
+                self._ipyw_sid.value = 'pootaasdsdfg'
```

Comment:
what is `poot...`? same for line 184

Reply:
oops. an old debugging artifact :)
killed

Reply:
you haven't!

Reply:
yes i have 
(forgot to commit - now should be ok...)